src/Content/OEmbed.php

```diff
@@ -315,10 +314,7 @@ private static function iframe($src, $width, $height)
 		}
 		$width = '100%';
 
-		// Only proxy OEmbed URLs to avoid mixed-content errors

```

Comment:
Mixed content problems cannot exist anymore?

Reply:
No because we're just pasting the Oembed HTML snippet directly into the post rendered HTML.